### ISSUE TITLE
Update to latest web5 packages

### ISSUE THREAD

**Issue by jiyoontbd** [TBD54566975/tbdex-js#160]
waiting on a few PRs in https://github.com/TBD54566975/web5-js/pulls to make changes to `@web5/dids` which will be compatible with the latest version of `@web5/credentials`

**Comment by jiyoontbd** [comment]
closing - duplicate of #162
